src/spec-configuration/containerFeaturesConfiguration.ts

```diff
@@ -926,19 +925,19 @@ async function applyFeatureConfigToFeature(output: Log, featureSet: FeatureSet,
 	const jsonString: Buffer = await readLocalFile(innerJsonPath);
 	const featureJson = jsonc.parse(jsonString.toString());
 
-	// TODO: Use spread operator {..., } to avoid needing to explicitly set each property.
 
-	feature.containerEnv = featureJson.containerEnv;
-	feature.buildArg = featureJson.buildArg;
-	feature.options = featureJson.options;
-	feature.installAfter = featureJson.installAfter;
-	feature.init = featureJson.init;
-	feature.privileged = featureJson.privileged;
-	feature.capAdd = featureJson.capAdd;
-	feature.securityOpt = featureJson.securityOpt;
-	feature.mounts = featureJson.mounts;
-	feature.entrypoint = featureJson.entrypoint;
+	feature = {
+		...featureJson,
+		...feature
+	};
+
+	let updatedFeature = updateFromOldProperties({ features: [feature] }).features[0];
 
+	feature = {
+		...updatedFeature
+	};
```

Comment:
I don't think this is any need to re-assign `feature` here AND manually assign it to `featureSet.feature[0]` below